src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -334,6 +337,15 @@ public ECPoint[] GetNextBlockValidators(StoreView snapshot)
                 .ToArray();
         }
 
+        public ECPoint[] GetNextNextBlockValidators(StoreView snapshot)
```

Comment:
Shouldn't we change `GetNextBlockValidators`, though?

Reply:
`GetNextBlockValidators` is also used in other places so we should not change this func directly